app/helpers/application_helper.rb

```diff
@@ -1,11 +1,11 @@
 module ApplicationHelper
-  include_concern 'ViewsShared'
-  include_concern 'Flash'
-  include_concern 'Listnav'
-  include_concern 'Navbar'
-  include_concern 'PageLayouts'
-  include_concern 'Tasks'
-  include Sandbox
+  include ApplicationHelper::ViewsShared
+  include ApplicationHelper::Flash
+  include ApplicationHelper::Listnav
+  include ApplicationHelper::Navbar
+  include ApplicationHelper::PageLayouts
+  include ApplicationHelper::Tasks
```

Comment:
Is the prefixing necessary? Or can you do `include ViewsShared`.  Note I actually prefer this explicit way, but I wasn't sure if it was forced to be this way.

Reply:
I think the idea is this avoids needing to use include_concern

Reply:
(regardless if zeitwerk fixes the use case from include_concern)